base/src/main/java/com/smartdevicelink/proxy/rpc/Show copy.java

```diff
@@ -0,0 +1,681 @@
+/*
+ * Copyright (c) 2017 - 2019, SmartDeviceLink Consortium, Inc.

```

Comment:
Please remove this copied file

Reply:
Please note this PR is currently not ready for review.